tests/Spout/Writer/XLSX/SheetTest.php

```diff
@@ -76,11 +77,22 @@ public function testSetSheetNameShouldThrowWhenNameIsAlreadyUsed()
     }
 
     /**
+     * @return void
+     */
+    public function testSetSheetVisibilityShouldCreateSheetHidden()
+    {
+        $fileName = 'test_set_visibility_should_create_sheet_hidden.xlsx';
+        $this->writeDataToHiddenSheet($fileName);
+
+        $this->assertIsHiddenSheet( $fileName, "The sheet visibility should have been changed to 'hidden'");
```

Comment:
rename to `assertContainsHiddenSheet` + remove space between `(` and `$fileName`